modules/ROOT/design-async-api.adoc

```diff
@@ -0,0 +1,126 @@
+= AsyncAPI Specifications   
+
+AsyncAPI is a language that describes messaging interfaces. This open-source, industry-standard language is agnostic to any technology. To learn more, see the https://www.asyncapi.com/docs/getting-started[AsyncAPI documentation]. 
+
+You can create or import your AsyncAPI specifications in API Designer and then edit, document, and publish them. 
+
+AsyncAPI makes it easy to work with event-driven architecture by separating the API into three layers:
+
+* Events: Message or data to be shared with other services
+* Channels: Destination of the message to be sent or received
+* Transport: Technology that transports the message, such as RabbitMQ, Kafka, or Anypoint MQ
+
+You can create or import an API specification in API Designer.
+
+To create an AsyncAPI specification in API Designer:
+
+. In Design Center, click *Create New*.
+. Select *New AsyncAPI*.
++
+image::platform/anypoint-create-asyncapi.png[The Create menu with the AsyncAPI option shown]
+
+== Import AsyncAPI Specifications
+
+To import AsyncAPI specifications from Anypoint Exchange or from your filesystem:
+
+. Click the gear icon at the top right of the text editor.
+. Select from the following options:
++
+** Select *Import from Exchange* to see the following lists:
+*** API specifications that are available from the business organization that your user ID belongs to in Anypoint Platform
+*** API specifications that are published by MuleSoft
+** Select *Import* to import an API specification from your local filesystem.
++
+image::platform/apid-import.png[The Settings menu with the Import options shown]
```

Comment:
I suggest removing this screenshot. It probably isn't needed. 

As long as the procedure and screenshot are still valid, it doesn't hurt to go ahead and publish like this for now to get it out there and get feedback. Up to you, of course.